lib/biran/config_defaults.rb

```diff
@@ -47,23 +47,35 @@ def config_dir
 
     def local_config_file
       ENV['BIRAN_LOCAL_CONFIG_FILE'] ||
-        File.join(app_shared_dir, configuration.config_dirname, configuration.local_config_filename)
+        File.join(app_shared_dir, configuration.config_dirname, local_config_filename)
+    end
+
+    def local_config_filename
+      ENV['BIRAN_LOCAL_CONFIG_FILENAME'] || app_config_defaults[:app][:local_config_filename] || configuration.local_config_filename
     end
 
     def vhost_public_dirname
       ENV['BIRAN_VHOST_PUBLIC_DIRNAME'] || app_config_defaults[:app][:vhost_public_dirname]
     end
 
     def db_config_override_file
-      File.join(app_shared_dir, configuration.config_dirname, configuration.db_config_filename)
+      File.join(app_shared_dir, configuration.config_dirname, db_config_filename)
```

Comment:
Wondering if all this `**_file` methods should be renamed to be `**_file_path` to better indicate the difference between the method that has the path and the method that is just the filename. Shouldn't be too much to refactor this as they should all be mostly internal methods I think.
Thoughts?